Sources/Brave/Frontend/Settings/Shields/OptionToggleView.swift

```diff
@@ -13,7 +13,7 @@ struct OptionToggleView: View {
   @ObservedObject var option: Preferences.Option<Bool>
   let onChange: ShieldToggleView.OnChangeCallback?
   
-  init(title: String, subtitle: String?,
+  init(title: String, subtitle: String? = nil,
```

Comment:
Why did you make this default value? I think it should just pass `nil` wherever it is used instead.

Reply:
It is a View which has title and subtitle, I think like tableview cell it should be default nil bit as far as I checked the usage in the app, most of the parts this is implemented there is subtitle so you are right about it.

Fix at https://github.com/brave/brave-ios/pull/7868/commits/01cf106574a2841cbd825ca0e37254f0a0f7e08f